Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -36,6 +37,7 @@ class AgendaDetailFragment : Fragment() {
 
     private lateinit var scheduleDetail: ScheduleDetail
     private lateinit var scheduleRowItem: ScheduleRow
+    private val speakerEvents = HashMap<String, SpeakerEvent>()
```

Comment:
probably really speakerDetails...not sure why that class was named "SpeakerEvent".  "EventSpeaker" would have made more sense.
Just commentary..you don't have to change it :-)